PayloadPkg/OsLoader/LoadImage.c

```diff
@@ -355,7 +355,15 @@ GetTraditionalLinux (
   }

```

Comment:
Do we need support the case where there is no config.cfg file but having kernel and initrd?
Remember previously we support this case and Yocto image doesn't have config.cfg file.

Reply:
OK, will consider to add this support.